cropclassification/helpers/config_helper.py

```diff
@@ -154,85 +158,36 @@ def parse_sensordata_to_use(input) -> Dict[str, SensorData]:
     return result
 
 
-def _get_raster_profiles() -> Dict[str, ImageProfile]:
+def _get_image_profiles(config_path: Path) -> Dict[str, ImageProfile]:
```

Comment:
```suggestion
def _get_image_profiles(image_profiles_path: Path) -> Dict[str, ImageProfile]:
```
Maybe clearer not to use the same variable name as for the general config file?